src/store/apps.js

```diff
@@ -11,6 +11,7 @@ const state = {
     edit: false
   },
   extensions: {},
+  fileSideBars: {},
```

Comment:
if the fileSideBar is the only planned sidebar, which we like to enhance with additional tabs; This data structure is sufficient.